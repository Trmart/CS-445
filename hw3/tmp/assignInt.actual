.   .   .   Child: 0  Id: gx of type undefined [line: 10]
.   .   .   Child: 0  Id: gx of type undefined [line: 9]
.   .   .   .   Child: 0  Id: gy of type undefined [line: 11]
.   .   .   .   Child: 0  Id: gy of type undefined [line: 12]
.   .   .   Child: 0  Id: px of type undefined [line: 18]
.   .   .   Child: 0  Id: px of type undefined [line: 19]
.   .   .   .   Child: 0  Id: py of type undefined [line: 20]
.   .   .   .   Child: 0  Id: py of type undefined [line: 21]
.   .   .   Child: 0  Id: sx of type undefined [line: 22]
.   .   .   Child: 0  Id: sx of type undefined [line: 23]
.   .   .   .   Child: 0  Id: sy of type undefined [line: 24]
.   .   .   .   Child: 0  Id: sy of type undefined [line: 25]
.   .   .   Child: 0  Id: x of type undefined [line: 13]
.   .   .   Child: 0  Id: x of type undefined [line: 14]
.   .   .   .   Child: 0  Id: y of type undefined [line: 15]
.   .   .   .   Child: 0  Id: y of type undefined [line: 16]
.   .   .   Child: 0  Id: zx of type undefined [line: 33]
.   .   .   Child: 0  Op: [ of type undefined [line: 11]
.   .   .   Child: 0  Op: [ of type undefined [line: 12]
.   .   .   Child: 0  Op: [ of type undefined [line: 15]
.   .   .   Child: 0  Op: [ of type undefined [line: 16]
.   .   .   Child: 0  Op: [ of type undefined [line: 20]
.   .   .   Child: 0  Op: [ of type undefined [line: 21]
.   .   .   Child: 0  Op: [ of type undefined [line: 24]
.   .   .   Child: 0  Op: [ of type undefined [line: 25]
.   Child: 0  Parm: px of type int [line: 4]
.   .   Child: 0  Var: x of type int [line: 6]
.   .   Child: 0  Var: zx of type int [line: 32]
.   .   Child: 1  Assign: <= [line: 9]
.   .   Child: 1  Call: esperanza [line: 33]
.   Child: 1  Compound [line: 31]
.   Child: 1  Compound [line: 5]
.   .   .   Child: 1  Const 111 of type int [line: 9]
.   .   .   Child: 1  Const 222 of type int [line: 11]
.   .   .   Child: 1  Const 333 of type int [line: 13]
.   .   .   Child: 1  Const 444 of type int [line: 15]
.   .   .   Child: 1  Const 555 of type int [line: 18]
.   .   .   Child: 1  Const 666 of type int [line: 20]
.   .   .   Child: 1  Const 777 of type int [line: 22]
.   .   .   Child: 1  Const 888 of type int [line: 24]
.   .   .   .   Child: 1  Const 9 of type int [line: 11]
.   .   .   .   Child: 1  Const 9 of type int [line: 12]
.   .   .   .   Child: 1  Const 9 of type int [line: 15]
.   .   .   .   Child: 1  Const 9 of type int [line: 16]
.   .   .   .   Child: 1  Const 9 of type int [line: 20]
.   .   .   .   Child: 1  Const 9 of type int [line: 21]
.   .   .   .   Child: 1  Const 9 of type int [line: 24]
.   .   .   .   Child: 1  Const 9 of type int [line: 25]
ERROR(10): Symbol 'output' is not declared.
ERROR(12): Symbol 'output' is not declared.
ERROR(14): Symbol 'output' is not declared.
ERROR(16): Symbol 'output' is not declared.
ERROR(17): Symbol 'outnl' is not declared.
ERROR(18): Symbol 'px' is not declared.
ERROR(19): Symbol 'output' is not declared.
ERROR(21): Symbol 'output' is not declared.
ERROR(23): Symbol 'output' is not declared.
ERROR(25): Symbol 'output' is not declared.
ERROR(26): Symbol 'outnl' is not declared.
Number of errors: 11
Number of warnings: 6
.   .   Sibling: 10  Call: output [line: 19]
.   .   Sibling: 11  Assign: <= [line: 20]
.   .   Sibling: 12  Call: output [line: 21]
.   .   Sibling: 13  Assign: <= [line: 22]
.   .   Sibling: 14  Call: output [line: 23]
.   .   Sibling: 15  Assign: <= [line: 24]
.   .   Sibling: 16  Call: output [line: 25]
.   .   Sibling: 17  Call: outnl [line: 26]
.   .   Sibling: 1  Call: output [line: 10]
.   .   .   Sibling: 1  Id: zy of type undefined [line: 33]
.   Sibling: 1  Parm: py of array of type int [line: 4]
Sibling: 1  Var: gy of array of type int [line: 2]
.   .   Sibling: 1  Var: y of array of type int [line: 6]
.   .   Sibling: 1  Var: zy of array of type int [line: 32]
.   .   Sibling: 2  Assign: <= [line: 11]
Sibling: 2  Func: esperanza returns type void [line: 4]
.   .   Sibling: 2  Var: sx of static type int [line: 7]
.   .   Sibling: 3  Call: output [line: 12]
Sibling: 3  Func: main returns type void [line: 30]
.   .   Sibling: 3  Var: sy of static array of type int [line: 7]
.   .   Sibling: 4  Assign: <= [line: 13]
.   .   Sibling: 5  Call: output [line: 14]
.   .   Sibling: 6  Assign: <= [line: 15]
.   .   Sibling: 7  Call: output [line: 16]
.   .   Sibling: 8  Call: outnl [line: 17]
.   .   Sibling: 9  Assign: <= [line: 18]
Var: gx of type int [line: 2]
WARNING(32): The variable 'zx' seems not to be used.
WARNING(32): The variable 'zy' seems not to be used.
WARNING(6): The variable 'x' seems not to be used.
WARNING(6): The variable 'y' seems not to be used.
WARNING(7): The variable 'sx' seems not to be used.
WARNING(7): The variable 'sy' seems not to be used.
