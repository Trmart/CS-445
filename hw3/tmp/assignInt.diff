1,14c1,86
< ERROR(10): Symbol 'output' is not declared.
< ERROR(12): Symbol 'output' is not declared.
< ERROR(14): Symbol 'output' is not declared.
< ERROR(16): Symbol 'output' is not declared.
< ERROR(17): Symbol 'outnl' is not declared.
< ERROR(19): Symbol 'output' is not declared.
< ERROR(21): Symbol 'output' is not declared.
< ERROR(23): Symbol 'output' is not declared.
< ERROR(25): Symbol 'output' is not declared.
< ERROR(26): Symbol 'outnl' is not declared.
< Number of errors: 10
< Number of warnings: 2
< WARNING(33): Variable 'zx' may be uninitialized when used here.
< WARNING(33): Variable 'zy' may be uninitialized when used here.
---
> .   .   .   Child: 0  Id: gx [line: 10]
> .   .   .   Child: 0  Id: gx [line: 9]
> .   .   .   .   Child: 0  Id: gy [line: 11]
> .   .   .   .   Child: 0  Id: gy [line: 12]
> .   .   .   Child: 0  Id: px [line: 18]
> .   .   .   Child: 0  Id: px [line: 19]
> .   .   .   .   Child: 0  Id: py [line: 20]
> .   .   .   .   Child: 0  Id: py [line: 21]
> .   .   .   Child: 0  Id: sx [line: 22]
> .   .   .   Child: 0  Id: sx [line: 23]
> .   .   .   .   Child: 0  Id: sy [line: 24]
> .   .   .   .   Child: 0  Id: sy [line: 25]
> .   .   .   Child: 0  Id: x [line: 13]
> .   .   .   Child: 0  Id: x [line: 14]
> .   .   .   .   Child: 0  Id: y [line: 15]
> .   .   .   .   Child: 0  Id: y [line: 16]
> .   .   .   Child: 0  Id: zx [line: 33]
> .   .   .   Child: 0  Op: [ of type undefined [line: 11]
> .   .   .   Child: 0  Op: [ of type undefined [line: 12]
> .   .   .   Child: 0  Op: [ of type undefined [line: 15]
> .   .   .   Child: 0  Op: [ of type undefined [line: 16]
> .   .   .   Child: 0  Op: [ of type undefined [line: 20]
> .   .   .   Child: 0  Op: [ of type undefined [line: 21]
> .   .   .   Child: 0  Op: [ of type undefined [line: 24]
> .   .   .   Child: 0  Op: [ of type undefined [line: 25]
> .   Child: 0  Parm: px of type int [line: 4]
> .   .   Child: 0  Var: x of type int [line: 6]
> .   .   Child: 0  Var: zx of type int [line: 32]
> .   .   Child: 1  Assign: <= [line: 9]
> .   .   Child: 1  Call: esperanza [line: 33]
> .   Child: 1  Compound [line: 31]
> .   Child: 1  Compound [line: 5]
> .   .   .   Child: 1  Const 111 of type int [line: 9]
> .   .   .   Child: 1  Const 222 of type int [line: 11]
> .   .   .   Child: 1  Const 333 of type int [line: 13]
> .   .   .   Child: 1  Const 444 of type int [line: 15]
> .   .   .   Child: 1  Const 555 of type int [line: 18]
> .   .   .   Child: 1  Const 666 of type int [line: 20]
> .   .   .   Child: 1  Const 777 of type int [line: 22]
> .   .   .   Child: 1  Const 888 of type int [line: 24]
> .   .   .   .   Child: 1  Const 9 of type int [line: 11]
> .   .   .   .   Child: 1  Const 9 of type int [line: 12]
> .   .   .   .   Child: 1  Const 9 of type int [line: 15]
> .   .   .   .   Child: 1  Const 9 of type int [line: 16]
> .   .   .   .   Child: 1  Const 9 of type int [line: 20]
> .   .   .   .   Child: 1  Const 9 of type int [line: 21]
> .   .   .   .   Child: 1  Const 9 of type int [line: 24]
> .   .   .   .   Child: 1  Const 9 of type int [line: 25]
> ERROR(18): Symbol 'px' is not declared.
> Number of errors: 1
> Number of warnings: 8
> .   .   Sibling: 10  Call: output [line: 19]
> .   .   Sibling: 11  Assign: <= [line: 20]
> .   .   Sibling: 12  Call: output [line: 21]
> .   .   Sibling: 13  Assign: <= [line: 22]
> .   .   Sibling: 14  Call: output [line: 23]
> .   .   Sibling: 15  Assign: <= [line: 24]
> .   .   Sibling: 16  Call: output [line: 25]
> .   .   Sibling: 17  Call: outnl [line: 26]
> .   .   Sibling: 1  Call: output [line: 10]
> .   .   .   Sibling: 1  Id: zy [line: 33]
> .   Sibling: 1  Parm: py of array of type int [line: 4]
> Sibling: 1  Var: gy of array of type int [line: 2]
> .   .   Sibling: 1  Var: y of array of type int [line: 6]
> .   .   Sibling: 1  Var: zy of array of type int [line: 32]
> .   .   Sibling: 2  Assign: <= [line: 11]
> Sibling: 2  Func: esperanza returns type void [line: 4]
> .   .   Sibling: 2  Var: sx of static type int [line: 7]
> .   .   Sibling: 3  Call: output [line: 12]
> Sibling: 3  Func: main returns type void [line: 30]
> .   .   Sibling: 3  Var: sy of static array of type int [line: 7]
> .   .   Sibling: 4  Assign: <= [line: 13]
> .   .   Sibling: 5  Call: output [line: 14]
> .   .   Sibling: 6  Assign: <= [line: 15]
> .   .   Sibling: 7  Call: output [line: 16]
> .   .   Sibling: 8  Call: outnl [line: 17]
> .   .   Sibling: 9  Assign: <= [line: 18]
> Var: gx of type int [line: 2]
> WARNING(2): The variable 'gx' seems not to be used.
> WARNING(2): The variable 'gy' seems not to be used.
> WARNING(32): The variable 'zx' seems not to be used.
> WARNING(32): The variable 'zy' seems not to be used.
> WARNING(6): The variable 'x' seems not to be used.
> WARNING(6): The variable 'y' seems not to be used.
> WARNING(7): The variable 'sx' seems not to be used.
> WARNING(7): The variable 'sy' seems not to be used.
