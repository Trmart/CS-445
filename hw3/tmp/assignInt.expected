ERROR(10): Symbol 'output' is not declared.
ERROR(12): Symbol 'output' is not declared.
ERROR(14): Symbol 'output' is not declared.
ERROR(16): Symbol 'output' is not declared.
ERROR(17): Symbol 'outnl' is not declared.
ERROR(19): Symbol 'output' is not declared.
ERROR(21): Symbol 'output' is not declared.
ERROR(23): Symbol 'output' is not declared.
ERROR(25): Symbol 'output' is not declared.
ERROR(26): Symbol 'outnl' is not declared.
Number of errors: 10
Number of warnings: 2
WARNING(33): Variable 'zx' may be uninitialized when used here.
WARNING(33): Variable 'zy' may be uninitialized when used here.
